src/Plugin.php

```diff
@@ -266,31 +266,73 @@ public static function getBaseUrl() {
    * Loads Google Analytics Data Layer related scripts.
    */
   public static function enqueueGaDataLayerScripts() {
+    global $wp;
+
     $handle = Plugin::PREFIX . '_datalayer';
     $scripts = static::getBaseUrl() . '/dist/scripts/datalayer';
+    $checkout_step_prefix = 'shop_analytics_checkout_step_';
+    $checkout_step = 0;
+    $wc_checkout_steps = apply_filters('shop_analytics_checkout_steps', [
+      'view_cart' => 1,
+      'checkout_page' => 2,
+      'checkout_pay_page' => 3,
+    ]);
+    $wc_endpoints = apply_filters('shop_analytics_wc_endpoints', [
+      'view-order' => 10,
+      'edit-account' => 20,
+      'edit-address' => 30,
+      'lost-password' => 40,
+      'customer-logout' => 50,
+      'add-payment-method' => 60,
+      'order-pay' => 4,
+      'order-received' => 5,
```

Comment:
why not keep the step of 10?

Reply:
I keep these values as defined by WooCommerce.